app/controllers/api/projects.php

```diff
@@ -603,6 +605,37 @@
         $response->dynamic($project, Response::MODEL_PROJECT);
     });
 
+App::patch('/v1/projects/:projectId/auth/session-alerts')
+    ->desc('Update project sessions emails')
+    ->groups(['api', 'projects'])
+    ->label('scope', 'projects.write')
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'projects')
+    ->label('sdk.method', 'updateSessionAlerts')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_PROJECT)
+    ->param('projectId', '', new UID(), 'Project unique ID.')
+    ->param('sessionAlerts', false, new Boolean(true), 'Set to true to enable session emails.')
```

Comment:
`enabled` might be a better name here, since the scope is already session alerts

Reply:
I looked the same direction, but in other endpoints we do this too. like `updateAuthLimit(limit)` or `updateAuthDuration(duration)` or `updateAuthStatus(status)`.
If you are against this pattern, we can make task in 1.6 QA board and discuss, but for now lets follow what is there.

With that in mind, I am changing it from `updateSessionAlerts(sessionAlerts)` to `updateSessionAlerts(alerts)`.